Makefile

```diff
@@ -1,24 +1,37 @@
-# USE_RESERVED=${USE_RESERVED:-false}
-# DEPLOY_NAME=${DEPLOY_NAME:-cnftestbed}
-# PACKET_FACILITY=${PACKET_FACILITY:-sjc1}
-# PACKET_OS=${PACKET_OS:-ubuntu_16_04}
-# MASTER_COUNT=${MASTER_COUNT:-1}
-# WORKER_COUNT=${WORKER_COUNT:-1}
-# PACKET_FACILITY=${PACKET_FACILITY:-sjc1}
-# VLAN_SEGMENT=${VLAN_SEGMENT:-$DEPLOY_NAME}
-# PLAYBOOK=${PLAYBOOK:-k8s_worker_vswitch_quad_intel.yml}
-# RELEASE_TYPE=${RELEASE_TYPE:-stable}
-# HOSTS_FILE=${HOSTS_FILE:-$(pwd)/data/$DEPLOY_NAME/nodes.env}
+#Vars req for running
+export PACKET_AUTH_TOKEN=YOUR_PACKET_AUTH_TOKEN
+export PROJECT_ID=YOUR_PACKET_PROJECT_ID
+export PACKET_PROJECT_NAME=YOUR_PACKET_PROJECT_NAME
```

Comment:
These exports probably needs to be removed. I was unable to run with the env vars set outside of the Makefile, when they were also present in here.

Reply:
Resolved by:
https://github.com/cncf/cnf-testbed/commit/ccf22f5ad60194dc14d2273f96525a5f74fdff5d